src/pages/Register.js

```diff
@@ -105,11 +159,81 @@ const Register = () => {
                 id,
                 value: email,
                 onChange: (v) => setEmail(v),
-                style: { width: '50ex' },
+                style: { width: '66ex' },
+              }),
+            ]),
+          ]),
+      ]),
+      h(IdContainer, [
+        (id) =>
+          div({ style: { lineHeight: '170%' } }, [
+            label({ htmlFor: id, style: { display: 'block', marginTop: '2rem' } }, ['Organization *']),
+            div([
+              h(TextInput, {
+                id,
+                value: institute,
+                onChange: (v) => setInstitute(v),
+                disabled: !partOfOrganization,
+                style: { width: '66ex' },
               }),
             ]),
           ]),
       ]),
+      h(IdContainer, [
+        (id) =>
+          h(div, { style: { lineHeight: '170%', style: { marginTop: '0.25rem' } } }, [
+            h(
+              LabeledCheckbox,
+              {
+                checked: !partOfOrganization,
+                onChange: () => setPartOfOrganization(!partOfOrganization),
+                'aria-describedby': id,
+              },
+              [label({ htmlFor: id, style: { marginLeft: '0.25rem' } }, ['I am not a part of an organization'])]
```

Comment:
Somehow this is working and not producing an error, but it is funky because you've never actually assigned `id` to any element. There _is_ no element-24 here. 

You can simplify this and remove the `IdContainer`. `LabeledCheckbox` should do the work for you… it creates an `IdContainer` itself to link the checkbox and the child items, which are the label.

<img width="990" alt="image" src="https://github.com/DataBiosphere/terra-ui/assets/484484/1fddb759-1f8a-4680-859b-42b1155a4b8c">

Reply:
Ah, good to know! Removed and the tests still pass and I don't see errors!